docs/provider/provider-tls.md

```diff
@@ -0,0 +1,361 @@
+
+| Record Type       | Host | Value |
+|-------------------|------|-------|
+| A-Record          | *    | Your-Server-Public-IP-Address-Here |
+
+</details>
+<br />
+
+### 📂 Update and Install Required Packages 
+
+On the server which you would like to turn into a provider, run updates and install the following packages:
+
+```bash
+sudo apt-get update
+sudo apt install certbot net-tools nginx python3-certbot-nginx -y
+```
+
+This guide uses `nginx` to handle routing to different provider processes, but you can also use Caddy, Envoy or your preferred solution.
+
+<br />
+
+### 📮 Generate Certificate 
+
+Next, we need to actually create the `TLS certificate` via the certifying authority. This process is automated by `certbot`.
+
+
+Use `certbot` to create a certificate:
+
+```
+sudo certbot certonly -d you.xyz -d lava.you.xyz -d eth.you.xyz
+```
+
+Note, you will need one `-d` flag for each subdomain you created as an `A-Record`. Even if you opted to create a Single Record, you still need to indicate a subdomain for each provider process you will run. We use the filler `you.xyz` as an example above.
+
+You may be met with several prompts. Use `nginx` or Nginx Web Server Plugin when asked. 
+<br />
+
+### 💻 Validate Certificate
+
+Let's make sure your certificate successfully installed! ✅ Input the following command:
+
+```sudo certbot certificates```
+
+Keep track of your output. If your certificate generation was successful, it should look as following:
+
+```
+Found the following certs:
+  Certificate Name: your-site.com
+    Domains: your-site.com eth.your-site.com lava.your-site.com
+    Expiry Date: 2023-11-07 14:37:29+00:00 (VALID: 84 days)
+    Certificate Path: /etc/letsencrypt/live/your-site.com/fullchain.pem
+    Private Key Path: /etc/letsencrypt/live/your-site.com/privkey.pem
+
+```
+
+You'll need both `Certificate Path` and `Private Key Path` for your next step.
+
+<br />
+
+### 🗃️ Add an Nginx Config for Each Domain
+
+Lava recommends running each chain under a separate provider process. This will separate error logs and protect against complete provider failure in the case of a problematic provider process. The first step of this is to create different nginx routes for each chain.
+
+For each chain that you want to support, you will need to create a separate `nginx` config file. 
+`cd` into `/etc/sites-available` and create a `server` file for each chain. You will need to select an open port for each chain. `Nginx` will use these config files to create your routes.
+
+<Tabs>
+<TabItem value="eth_ex" label="eth_server">
+
+🟢 ```sudo nano eth_server``` 
+
+
+```
+server {
+    listen 443 ssl http2;
+    server_name eth.your-site.com;
+
+    ssl_certificate /etc/letsencrypt/live/your-site.com/fullchain.pem;
+    ssl_certificate_key /etc/letsencrypt/live/your-site.com/privkey.pem;
+    error_log /var/log/nginx/debug.log debug;
+
+    location / {
+        proxy_pass http://127.0.0.1:2223;
+        grpc_pass 127.0.0.1:2223;
+    }
+}
```

Comment:
Is there a reason to not have a redirect from port 80 to 443?

Reply:
I'm not certain- I think @ranlavanet had a reason but he's OOO. What do you suggest?

Reply:
Well, it was stated that secure connections is mandatory, but it is customary to have an "upgrade" from insecure to secure connection (3xx redirect from :80 to :443). Not critical, but nice to have. We can move forward without it.

Reply:
Purely speculation, but may have something to do with how Nginx handles insecure requests. TBD ✅